relay-server/src/utils/rate_limits.rs

```diff
@@ -256,7 +256,8 @@ impl EnvelopeSummary {
 
 /// Rate limiting information for a data category.
 #[derive(Debug)]
-struct CategoryLimit {
+#[cfg_attr(test, derive(Clone))]
```

Comment:
I wanted to have the `clone()` in tests. Either we implement manually the `Clone` in the tests or we go this route. Idk which would be preferred. The manual definition might be more verbose.

Reply:
Fine with me!